using-mobile-extensions/adobe-journey-optimizer/api-reference.md

```diff
@@ -1,5 +1,41 @@
 # Adobe Journey Optimizer API reference
 
+## addPushTrackingDetails (Android only) <a id="addpushtrackingdetails"></a>
+
+The `addPushTrackingDetails` API is used to update a pending intent with important information, such as messageId and Customer Journey information.
+
+**Note:** Calling this API is mandatory, so the pending intent can be used while tracking push notification interactions.
```

Comment:
Is there a reason we're not just using the built-in gitbook notes?

Reply:
no, this was just missed as these docs were migrated from the messaging extension repo docs. i'll fix this.